indico/modules/events/abstracts/templates/abstract/public.html

```diff
@@ -232,6 +232,46 @@
         </div>
 {% endmacro %}
 
+{% macro _render_judgment_action(abstract) %}
+    {% set action = abstract.state.name %}
+    {% if abstract.can_judge(session.user) %}
+        {% set judge_name = abstract.judge.name %}
```

Comment:
You can get in here if the abstract is e.g. withdrawn (which means there is no judge) since the macro is called whenever the state is not submitted and `can_judge` doesn't check the abstract state.